docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -540,8 +535,12 @@ Location named `demo1` with labels `foo=bar` and `the-word=the-bird`.
 This was equivalent to the following commands.
 
 ```shell
-kubectl ws root:imw-1
-kubectl create -f -<<EOF
+imw_space_config="${PWD}/temp-space-config/spaceprovider-default-imw-1"
+secret_name=$(kubectl --kubeconfig $sm_core_config --context $sm_context get space imw-1 -n spaceprovider-default -o jsonpath="{$.status.externalSecretRef.name}")
```

Comment:
What is `$sm_core_config`?

If this kubectl plugin is sensitive to unusual environment variables then that should be documented in the description of the command.